### DISCUSSION TITLE
Introducing Nazi Zombies: Portable Discord, Matrix, & LiberaChat Communities

### DISCUSSION THREAD

**Discussion by MotoLegacy** [nzp-team/nzportable#550] [Announcements]
What it says on the tin!.. I've launched a Discord community and registered an IRC channel for Nazi Zombies: Portable related discussion.

This community has a rough history with stability and sensibility when it comes to instant messaging platforms, so I've made collaborations to ensure these groups are a safe and regulated space. A lot of the nonsense you may have seen under previous attempts (and unofficial ones) will not fly here.

You can access the server via https://discord.nzp.gay/

And via #nzportable on irc.libera.chat

The Matrix space is now available through matrix.nzp.gay

A Matrix bridge will soon be arriving that will connect all three sources so everyone can chat in one place and be heard everywhere.

Consider becoming a [Patron](https://patreon.com/cypressimplex), thanks.

**Comment by MotoLegacy** [comment]
The Matrix space is now available through matrix.nzp.gay

**Reply by alexisruizleon** [reply]
Who ever is reading don't join they kick you and are not willing to help with people that need help

**Reply by katniny** [reply]
> Who ever is reading don't join they kick you and are not willing to help with people that need help

if you ask stupid questions, yes..
otherwise, no. i've seen a lot of people be extremely helpful to those who are asking good questions though, not "how do i put nazis on my iOS with my portable ps5 controller".
however, you can play NZ:P on your iPhone with [PPSSPP](https://apps.apple.com/us/app/ppsspp-psp-emulator/id6496972903), downloading the PSP version of [NZ:P](https://github.com/nzp-team/nzportable/releases/tag/nightly), then wirelessly connecting your controller via bluetooth (make sure to map the buttons if needed).

**Reply by Shmoster93** [reply]
> Who ever is reading don't join they kick you and are not willing to help with people that need help

That's hilarious! I just got kicked after asking for a map file that's not yet on the github. They dished me attitude and couldn't take any back. It's all in good fun, the mods just gotta relax. We just wanna play a free game

**Reply by MotoLegacy** [reply]
Overlook is distributed via GitHub wdym

**Reply by Shmoster93** [reply]
> Overlook is distributed via GitHub wdym

It's not in "map releases" so if theres a file of it floating around anywhere I've yet to find it. It's fine, I just downloaded the latest version of NZP which included it. Since i enjoy the game, I'd like updates on maps and map making tutorials, but now I'm locked out of the discord for no reason.

**Reply by MotoLegacy** [reply]
> I'd like updates on maps and map making tutorials, but now I'm locked out of the discord for no reason.

Discord isn't used for this anyway because the platform is, by design, gatekept to registered users. I have never locked those types of things to Discord.

mapping Docs @ https://docs.nzp.gay/

**Comment by alexisruizleon** [comment]
I meant on my “backbone “ I figured it how to download it by myself but the
buttons for shooting and looking around don’t work well there not stupid
questions you just gotta communicate not just randomly kick people lol you
weird for that

**Comment by MrWhoHacking** [comment]
same they got mad and upset..then look at my profile and just cause i play crypto games they ban me... i feel very sorry for the guy

**Reply by katniny** [reply]
you didnt mention all your demands then calling the owner an "ass" and "sad person" when she told you no

quite convenient that those details got left out!

**Reply by MrWhoHacking** [reply]
> you didnt mention all your demands then calling the owner an "ass" and "sad person" when she told you no
> 
> 
> 
> quite convenient that those details got left out!

That was after what he she said to me.  pull the logs and show the whole conversation lol. Anyways

**Reply by katniny** [reply]
i just know you'd be really fun to ragebait, but alas... i must practice self-control

**Reply by MrWhoHacking** [reply]
Right it's ok my friend. It's Just a server...you guys really  aren't any helpful and the beauty of ai they can help the user more then person who made it which it's  sad

**Comment by Mikethemlgdeck** [comment]
I had to figure out how to map the camera to the face buttons by myself since no one was replying or bothering to help me, one hour later a community dev responded with "It's not about handedness (normal response) and then after I said I had fixed it, they responded with "sorry to hear that you're just going to have to adjust" so I responded by saying it works better this way imo and then they just went "sorry to hear that" but even before that they were just going around telling people who were asking how to map camera controls to face buttons "just use analog, it's way better" or "just adapt to analog" completely dismissing them instead of helping them.

I went ahead and left the server after seeing that behavior and having that interaction with them, genuinely wouldn't recommend anyone joining the discord since they genuinely aren't helpful and will probably outright dismiss you.

Here's a couple images to show everyone what I mean.

<img width="1080" height="916" alt="Screenshot_20260911-202949_Discord" src="https://github.com/user-attachments/assets/c4d2121a-69fc-4617-8167-836155ec3f38" />
<img width="1080" height="698" alt="Screenshot_20260911-202936_Discord" src="https://github.com/user-attachments/assets/05703789-f5fa-439d-bb14-9c5e4c8a9340" />

**Reply by MotoLegacy** [reply]
i personally think its a totally valid response to just say sorry since you didn't wanna ask why it is the way it is, IMO. this was a pretty straightforward exchange
